Sources/MessageInputBar.swift

```diff
@@ -424,14 +424,14 @@ open class MessageInputBar: UIView {
     }
     
     open func textViewDidChange() {
-        let trimmedText = inputTextView.text.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
Was it necessary to remove this? I like how the send button is enabled/disabled by taking into account if a message only contains white space